appinventor/components/src/com/google/appinventor/components/runtime/LinkedData.java

```diff
@@ -107,9 +107,9 @@ public void EndpointURL(String url) {
    * @param query Query text to execute
    */
   @SimpleFunction
-  public void ExecuteSPARQLQuery(final String query) {
+  public void ExecuteSPARQLQuery(final Object query) {
```

Comment:
Why was this changed to Object from String? When would you pass anything other than a String?